mmv1/third_party/terraform/services/bigquery/resource_bigquery_table_test.go.erb

```diff
@@ -3626,6 +3757,143 @@ resource "google_bigquery_table" "test" {
 `, datasetID, tableID, schema)
 }
 
+<% unless version == 'ga' -%>
+func testAccBigQueryTableWithReplicationInfoAndView(datasetID, tableID string) string {
+	return fmt.Sprintf(`
+resource "google_bigquery_dataset" "test" {
+  provider = google-beta
+
+  dataset_id = "%s"
+}
+
+resource "google_bigquery_table" "test" {
+  provider = google-beta
+
+  deletion_protection = false
+  table_id   = "%s"
+  dataset_id = google_bigquery_dataset.test.dataset_id
+  view {
+    query          = "SELECT state FROM [lookerdata:cdc.project_tycho_reports]"
+    use_legacy_sql = true
+  }
+  table_replication_info {
+    source_project_id = "source_project_id"
+    source_dataset_id = "source_dataset_id"
+    source_table_id = "source_table_id"
+  }
+}
+`, datasetID, tableID)
+}
+
+func testAccBigQueryTableWithReplicationInfo(projectID, sourceDatasetID, sourceTableID, sourceMVID, replicaDatasetID, replicaMVID, sourceMVJobID, dropMVJobID, replicationIntervalExpr string) string {
+	return fmt.Sprintf(`
+resource "google_bigquery_dataset" "source" {
+  provider = google-beta
+
+  dataset_id = "%s"
+  location = "aws-us-east-1"
+}
+
+resource "google_bigquery_table" "source_table" {
+  provider = google-beta
+
+  deletion_protection = false
+  table_id   = "%s"
+  dataset_id = google_bigquery_dataset.source.dataset_id
+  external_data_configuration {
+    connection_id   = "bigquerytestdefault.aws-us-east-1.e2e_ccmv_tf_test"
```

Comment:
Is it ok to have this baked into the test, which could be run by any contributor, Googler or otherwise?

Reply:
Sadly a restriction of this feature as it depends on external resources. Similar setup was done in https://github.com/GoogleCloudPlatform/magic-modules/pull/9440.